config/settings.rb

```diff
@@ -7,6 +7,79 @@
 SETTINGS.merge! YAML.load(ERB.new(File.read("#{root}/#{settings_file}")).result) if File.exist?(settings_file)
 SETTINGS[:version] = Foreman::Version.new
 
+# Load settings from env variables
+ENV_VARIABLE_SETTINGS_KEYS = {
+  'FOREMAN_UNATTENDED' => [:boolean, :unattended],
+  'FOREMAN_REQUIRE_SSL' => [:boolean, :require_ssl],
+  'FOREMAN_SUPPORT_JSONP' => [:boolean, :support_jsonp],
+  'FOREMAN_MARK_TRANSLATED' => [:boolean, :mark_translated],
+  'FOREMAN_WEBPACK_DEV_SERVER' => [:boolean, :webpack_dev_server],
+  'FOREMAN_WEBPACK_DEV_SERVER_HTTPS' => [:boolean, :webpack_dev_server_https],
+  'FOREMAN_ASSETS_DEBUG' => [:boolean, :assets_debug],
+  'FOREMAN_HSTS_ENABLED' => [:boolean, :hsts_enabled],
+  'FOREMAN_RAILS' => [:boolean, :hsts_enabled],
```

Comment:
I'm having my doubts about this. Currently this must be set in `boot_settings` since that one is included in `Gemfile`. Setting it this late just breaks it if they don't match.

This at least needs to set the right variable :)

```suggestion
  'FOREMAN_RAILS' => [:boolean, :rails],
```

Reply:
Yeah, found that as well while writing the tests. ;-)